.github/workflows/release.yml

```diff
@@ -60,12 +64,24 @@ jobs:
         id: go
       - name: Check out code into the Go module directory
         uses: actions/checkout@v1
+      - name: Install Android NDK
+        run: |
+          curl -L https://dl.google.com/android/repository/android-ndk-r21e-linux-x86_64.zip -o android-ndk.zip
+          unzip android-ndk.zip -d $HOME/android-ndk
+          export ANDROID_NDK_HOME=$HOME/android-ndk/android-ndk-r21e
+          export PATH=$PATH:$ANDROID_NDK_HOME/toolchains/llvm/prebuilt/linux-x86_64/bin
```

Comment:
can we set these in the `env` section below?

Reply:
Android ndk needs to be set before building process so that gcc compiler can recognise the `arm-linux-androideabi-gcc` for building `arm` binaries.

Basically, support for Android/arm almost been a headache to me 😅